lib/reline/line_editor.rb

```diff
@@ -1133,6 +1135,9 @@ def input_key(key)
     else
       normal_char(key)
     end
+
+    @prev_action_state, @next_action_state = @next_action_state, NullActionState
```

Comment:
Can you add an initialization of these instance variables in `def reset_variables`?
This will properly re-initialize just ater Reline.readline is canceled by Ctrl-C.
Initial state of `@prev_action_state` will be simple. (Currently, initial value is nil and will be updated to NullActionState)

Reply:
Thanks for the advice. I have also added initialization of variables at 19dae5bdffd5e7debad46d0be8f3ee370fe8ec5b